components/InteriorLeftNavItem.js

```diff
@@ -40,9 +46,7 @@ const InteriorLeftNavItem = ({
       onKeyPress={evt => onClick(evt, href)}
       {...other}
     >
-      <a className="left-nav-list__item-link" href={href} tabIndex={tabIndex}>
-        {label}
-      </a>
+      {newChild}
```

Comment:
You could check if `children` is empty. I'd probably add a helper function:

```jsx
const getLink = (children, label) => {
  if (children) {
    // return the child
  }

  // return a generated link using the current pattern
};
```

You could deprecate the `label` approach if you wanted, maybe?